src/main/java/appeng/tile/misc/TileCellWorkbench.java

```diff
@@ -195,20 +195,21 @@ else if( this.manager.getSetting( Settings.COPY_MODE ) == CopyMode.CLEAR_ON_REMO
 		}
 		else if( inv == this.config && !this.locked )
 		{
-			final IInventory c = this.getCellConfigInventory();
+			final IItemHandlerModifiable c = this.getCellConfigInventory();
 			if( c != null )
 			{
-				for( int x = 0; x < this.config.getSizeInventory(); x++ )
-				{
-					c.setInventorySlotContents( x, this.config.getStackInSlot( x ) );
+				for( int x = 0; x < this.config.getSlots(); x++ )
+				{				
+					c.setStackInSlot( x, this.config.getStackInSlot( x ) );
 				}
 
-				c.markDirty();
+				//TODO: should not be needed anymore
+				//c.markDirty();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)